generators/kubernetes/templates/db/postgresql.yml.ejs

```diff
@@ -28,9 +28,12 @@ spec:
   resources:
     requests:
       storage: 2Gi
-  <%_ if (kubernetesStorageClassName) { _%>
+<%_ if (kubernetesStorageClassName && !minikube) { _%>
```

Comment:
`kubernetesStorageClassName` value can be set as `standard` in the logic rather than adding redundant condition here.